src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const valueStyles = sourceString.split(';')
+    .map(style => style.split(':')
+      .map(data => data.trim()))
+    .filter(style => style.length > 1);
```

Comment:
instead of two map and filter iterations consider to make everything in reduce method
```suggestion
  const valueStyles = sourceString.split(';')
```